utils/oath-toolkit/Makefile

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright (C) 2016 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=oath-toolkit
+PKG_VERSION:=2.6.2
+PKG_RELEASE:=1
+PKG_SOURCE:=oath-toolkit-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://download.savannah.gnu.org/releases/oath-toolkit/
+PKG_SOURCE_MD5SUM:=4a05cd4768764843bd5493609a6bdb17
+PKG_LICENSE:=GPL-3.0+
+PKG_LICENSE_FILES:=COPYING
+PKG_MAINTAINER:=Fam Zheng <fam@euphon.net>
+
+include $(INCLUDE_DIR)/package.mk
+
+CONFIGURE_ARGS += \
+	--disable-xmltest \
+	--disable-pskc
+
+define Package/oath-toolkit
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=Oath toolkit
```

Comment:
You need to provide better description about the package for others:

* The TITLE should offer short one-line explanation about the package to be shown in menuconfig. About 30 characters. E.g. ``` Toolkit for building one-time password authentication ```

* You should also write a ``` Package/oath-toolkit/description ``` section, where you have a bit longer explanation about the package. for example, the first paragraph from the oath-toolkit homepage:
``` The OATH Toolkit provide components for building one-time password authentication systems. It contains shared libraries, command line tools and a PAM module. Supported technologies include the event-based HOTP algorithm (RFC4226) and the time-based TOTP algorithm (RFC6238). OATH stands for Open AuTHentication, which is the organization that specify the algorithms. ```   (Note that I left the PKSC sentence out, as I think that you have disabled it in configure.)